circuits/types/field/src/lib.rs

```diff
@@ -156,6 +156,18 @@ impl<E: Environment> From<&Field<E>> for LinearCombination<E::BaseField> {
     }
 }
 
+impl<E: Environment> From<Field<E>> for Vec<Boolean<E>> {
+    fn from(field: Field<E>) -> Self {
+        field.to_bits_le()
+    }
+}
+
+impl<E: Environment> From<&Field<E>> for Vec<Boolean<E>> {
+    fn from(field: &Field<E>) -> Self {
+        field.to_bits_le()
+    }
+}
+
```

Comment:
Remove this and use `.to_bits_le()`. This call has no notion of endianness and should be avoided at all costs.